Wright_Will/Assignments/login_and _registration/templates/success.html

```diff
@@ -3,7 +3,7 @@
     <head>
         <meta charset="utf-8">
         <title>success</title>
-        <link rel="stylesheet" href="{{url_for('static',styles.css)}}" >
+        <link rel="stylesheet" href={{url_for('static',filename='styles.css')}} >
```

Comment:
These need to be encapsulated with quotes. If you filename has a space in it, you're screwed. For example, imagine if the filename is "my styles.css". This would render as follows:

`<link rel="stylesheet" href=static/my styles.css>`

Which is incorrect